modules/generative-cohere/clients/cohere.go

```diff
@@ -189,22 +190,23 @@ func (v *cohere) getApiKey(ctx context.Context) (string, error) {
 }
 
 type generateInput struct {
-	Prompt            string   `json:"prompt"`
-	Model             string   `json:"model"`
-	MaxTokens         int      `json:"max_tokens"`
-	Temperature       int      `json:"temperature"`
-	K                 int      `json:"k"`
-	StopSequences     []string `json:"stop_sequences"`
-	ReturnLikelihoods string   `json:"return_likelihoods"`
+	ChatHistory   []message `json:"chat_history"`
```

Comment:
please add `omitempty` annotation to the optional parameters. For example here optional chatHistory parameter should be annotated like so:
```
`json:"chat_history,omitempty"`
```